src/extras/primitives/primitives/a-camera.js

```diff
@@ -3,17 +3,16 @@ var registerPrimitive = require('../registerPrimitive');
 registerPrimitive('a-camera', {
   defaultAttributes: {
     camera: {},
-    'look-controls': {},
-    'wasd-controls': {}
+    controls: {}
   },
 
   mappings: {
     active: 'camera.active',
     far: 'camera.far',
     fov: 'camera.fov',
-    'look-controls-enabled': 'look-controls.enabled',
     near: 'camera.near',
-    'wasd-controls-enabled': 'wasd-controls.enabled'
+    'position-controls-enabled': 'controls.positionEnabled',
```

Comment:
I know it's a WIP, but caught that this isn't consistent with the docs and the line below.

I might keep it as `position-enabled` + `rotation-enabled` to be consistent with the prop name.